src/workspaceViewer.ts

```diff
@@ -110,9 +110,16 @@ export class WorkspaceItem extends TreeItem {
 			return str;
 		}
 	}
+
+	private getSizeString(bytes: number): string {
```

Comment:
There's also `format(object.size(x), units = 'auto')` in R, which might look a bit nicer

Reply:
Thanks for reminding! I thought it might be useful to keep passing `number` instead of `string` to keep the interface clean.

Reply:
Agree that having the number availabl in vscode can be useful (e.g. to sort by size), but I think having both `size` and `sizeString` would be worth the UI improvement. We could also try to replicate the `format(...)` behaviour in typescript, but I think using `format()` is somewhat cleaner

Reply:
Is there a notable difference between the result string from `format()` and `getSizeString()`? Looks like they are almost the same?

Reply:
`x <- 1` gives `56 bytes` vs `56.00  B` (with a double space).
`v <- as.list(1:10000)` gives `625 Kb` vs `625.05 KB`
In both cases I personally prefer the output of `format()`

Reply:
I see. I update the format of `getSizeString()`, and it returns exactly the same format now.